From 1dc7d51afb0cc8147ec8889fc7ae8ec56f1311b9 Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Thu, 16 Apr 2020 09:12:50 +0300
Subject: [PATCH 49/55] ARM: imx7d: release kernel v4.14.98-cl-som-imx7-6.2

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ddda0c8462e6..6a831a1b06ce 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 4
 PATCHLEVEL = 14
 SUBLEVEL = 98
-EXTRAVERSION = -cl-som-imx7-6.1
+EXTRAVERSION = -cl-som-imx7-6.2
 NAME = Petit Gorille
 
 # *DOCUMENTATION*
-- 
2.20.1

