From 49f98b74532cea30d34c7bc20dc01b58d004cfee Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Tue, 21 Apr 2020 15:53:23 +0300
Subject: [PATCH 52/55] ARM: imx7d: release kernel v4.14.98-cl-som-imx7-6.3

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 6a831a1b06ce..1d8204cee69a 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 4
 PATCHLEVEL = 14
 SUBLEVEL = 98
-EXTRAVERSION = -cl-som-imx7-6.2
+EXTRAVERSION = -cl-som-imx7-6.3
 NAME = Petit Gorille
 
 # *DOCUMENTATION*
-- 
2.20.1

